oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleConfig.java

```diff
@@ -134,6 +134,24 @@
     @Getter
     private String searchableLogsTags = "";
 
+    /**
+     * The number of threads used to synchronously refresh the metrics data to the storage. When the value is negative,
+     * the number of processors will used as the default value.
+     *
+     * @since 8.5.0
+     */
+    @Getter
+    @Setter
+    private int syncThreads = 10;
```

Comment:
If the user doesn't specify this, for example, use an old `application.yaml` file (without this key), the default thread count is 10, should it be processor count here too?